packages/eslint-config/index.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
     'no-var': 2,
     'prefer-const': 2,
     'prefer-destructuring': 1,
+    '@typescript-eslint/camelcase': ['warn'],
```

Comment:
Looks this rule is [deprecated](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/camelcase.md) and we should use [naming-convention](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/naming-convention.md) rule.